src/libradosstriper/RadosStriperImpl.cc

```diff
@@ -695,10 +695,12 @@ int libradosstriper::RadosStriperImpl::aio_generic_stat
     delete getxattr_completion;
     multi_completion->complete_request(rc);
     multi_completion->put();
+    delete multi_completion;
```

Comment:
@chipitsine (NIT) Please consider to keep PR title < 50 characters for future PRs
https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#title-of-pull-requests-and-title-of-commits